OpenRA.Mods.Common/Activities/FindResources.cs

```diff
@@ -53,10 +53,7 @@ public override Activity Tick(Actor self)
 				return NextActivity;
 
 			if (harv.IsFull)
-			{
-				// HACK: DeliverResources is ignored if there are queued activities, so discard NextActivity
-				return ActivityUtils.SequenceActivities(new DeliverResources(self));
-			}
+				return ActivityUtils.SequenceActivities(self, new DeliverResources(self), NextActivity);
```

Comment:
Note to self: Test this before approving.

Reply:
This does regress it to the pre-hack behavior, so in my opinion this change should be left to #16193.

Reply:
Yes, this does not belong in this PR. Reverted.